ci-operator/config/openshift/builder/openshift-builder-master.yaml

```diff
@@ -73,6 +73,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-cgroupsv2
+- as: e2e-aws-techpreview-featuregate
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-techpreview
```

Comment:
this is at least the correct workflow to start with from what I can tell @jkhelil 

but I suspect ultimately we will need to create a new workflow that merges openshift-e2e-aws-techpreview with 
openshift-e2e-aws-builds

we ultimately need a trigger of the build e2e's in openshift/origin

See https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/aws/builds/openshift-e2e-aws-builds-workflow.yaml#L12